src/rules/padding-before-describe-blocks.ts

```diff
@@ -1,48 +1,14 @@
-/**
- * @fileoverview Enforces single line padding around describe blocks
- * @author Dan Green-Leipciger
- */
-'use strict';
-
-import { padBefore } from '../utils';
-//------------------------------------------------------------------------------
-// Rule Definition
-//------------------------------------------------------------------------------
-
-const beforeMessage =
-  'You need a newline or comment before a describe block when it comes after another expression';
-
-const isDescribe = node =>
-  node.expression &&
-  node.expression.callee &&
-  node.expression.callee.name === 'describe';
+import padding from './padding';
 
 export default {
-  meta: {
-    docs: {
-      description: 'Enforces at least a line of padding before describe blocks',
-      category: 'Fill me in',
-      recommended: false,
-    },
-    fixable: 'whitespace', // or "code" or "whitespace"
-    schema: [
-      // fill in your schema
-    ],
-  },
-  beforeMessage,
+  ...padding,
```

Comment:
Here we take the generic `padding` rule and spread it, overwriting the `create` method below. In real life (once this is no longer a poc) we'd want to also override the `docs` portion of this rule.